staging/src/k8s.io/sample-apiserver/pkg/admission/initializer/initializer.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package initializer
+
+import (
+	"k8s.io/apiserver/pkg/admission"
+	informers "k8s.io/sample-apiserver/pkg/client/informers_generated/internalversion"
+)
+
+type pluginInitializer struct {
+	informers informers.SharedInformerFactory
+}
+
+// New creates an instance of admission plugins initializer.
+func New(informers informers.SharedInformerFactory) (pluginInitializer, error) {
```

Comment:
I want to keep it simple, all I need are informers.
The plugin I am going to implement will use the informers like this:

`SharedInformerFactory.Wardle().InternalVersion().Fischers().Lister().List`